src/wagtail_polymath/config.py

```diff
@@ -54,5 +55,25 @@
 def get_polymath_config(key):
     paths = POLYMATH_SETTINGS.get(key, [])
 
-    # Return absolute path to the asset if it's a static file path.
-    return [versioned_static(path) if finders.find(path) else path for path in paths]
```

Comment:
I think it would be good to have this URL validation, so I am adding validation logic for a generic URL.